diff --git a/model/feature_extractor.py b/model/feature_extractor.py
index 4ee752e..09c351c 100644
--- a/model/feature_extractor.py
+++ b/model/feature_extractor.py
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
diff --git a/model/supersimplenet.py b/model/supersimplenet.py
index 40b71a8..9c2fe79 100644
--- a/model/supersimplenet.py
+++ b/model/supersimplenet.py
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
 import math
 from collections import OrderedDict
 from pathlib import Path
